pawn/Elements/Player/Drink.pwn

```diff
@@ -56,14 +56,14 @@ CDrink__SetVars(bool:all = false)
     }
 
     // Alcohol units in each of the drinks
-    drinksData[0][0] = CDrink__RandAlcoholVolume(2, 3); // Wine         $2000
-    drinksData[1][0] = CDrink__RandAlcoholVolume(1, 2); // Beer         $1000
+    drinksData[0][0] = CDrink__RandAlcoholVolume(2, 3); // Wine         $3000
+    drinksData[1][0] = CDrink__RandAlcoholVolume(1, 2); // Beer         $2000
     drinksData[2][0] = CDrink__RandAlcoholVolume(1, 2); // Cider        $1000
     drinksData[3][0] = CDrink__RandAlcoholVolume(1, 3); // Alcopop      $2000
     drinksData[4][0] = CDrink__RandAlcoholVolume(2, 6); // Cocktail     $3000
     drinksData[5][0] = CDrink__RandAlcoholVolume(1, 1); // Shots        $1000
     drinksData[6][0] = CDrink__RandAlcoholVolume(5, 8); // Vodka        $5000
-    drinksData[7][0] = CDrink__RandAlcoholVolume(5, 8); // Whisky       $5000
+    drinksData[7][0] = CDrink__RandAlcoholVolume(5, 8); // Whisky       $4000
```

Comment:
Updating the comment won't do a lot :-). In fact, they are incorrect to start with.

The price of the drinks is decided by the random number ([5, 8] in this line) multiplied by DRINK_COSTPERUNIT, which is set to $1000. That means that the price will be between [$5000, $8000] for Whisky.

If you want to lower prices, I suggest that you change the DRINK_COSTPERUNIT constant on line 32 of this file to 750 or so. That will immediately affect all prices.

If you do that and want to be super awesome, it would be great if you could **remove** the prices listed at the end of lines 59-66 completely, as they are wrong.